client/my-sites/plan-features-2023-grid/components/actions.tsx

```diff
@@ -60,6 +60,11 @@ const DummyDisabledButton = styled.div`
 	text-align: center;
 `;
 
+// @todo npm-migration
+// getPlanBillPeriod from plans data store
+// pass domainFromHomeUpsellFlow as prop
```

Comment:
> pass domainFromHomeUpsellFlow as prop

I'm handling this one next. 
leads to some more general refactors in how we can pass and execute external actions